### PR TITLE
New Errors

### PR DESCRIPTION
**Pull request by jasonfellows**
* Update errors to allow passing in an array
* Errors have new style guide appearance
* Use full y-axis padding for typography p, small

### CODE REVIEW

**Comment by keithpops** [comment]
+1

File: src/js/components/forms/field-errors.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    });
+  },
+
+  render() {
+    if (this.props.errors.length > 0) {
+      return <div className={this.classes()}>
+        {this.list()}
+      </div>;
+    } else {
+      return <span />;
```
Comment (HipsterBrown) [inline src/js/components/forms/field-errors.js]: Why render an empty `span`?

Comment (jasonfellows) [inline src/js/components/forms/field-errors.js]: It was the most inert JSX component I could think of.

Comment (HipsterBrown) [inline src/js/components/forms/field-errors.js]: From the [React docs](http://facebook.github.io/react/docs/component-specs.html#render):
> You can also return null or false to indicate that you don't want anything rendered. 

Can we do this instead?

**Comment by HipsterBrown** [comment]
:+1:

**Pull request merged**